docs/fundamentals/runtime-libraries/snippets/System.Collections.Generic/ListT/Overview/csharp/program.cs

```diff
@@ -59,8 +59,7 @@ public static void Main()
 
         // Check the list for part #1734. This calls the IEquatable.Equals method
         // of the Part class, which checks the PartId for equality.
-        Console.WriteLine("\nContains(\"1734\"): {0}",
-        parts.Contains(new Part { PartId = 1734, PartName = "" }));
+        Console.WriteLine($"\nContains(\"1734\"): {parts.Contains(new Part { PartId = 1734, PartName = "" })}");
```

Comment:
This would be great with raw string literals:

```suggestion
        Console.WriteLine($"""
        Contains("1734"): {parts.Contains(new Part { PartId = 1734, PartName = "" })}
        """);
```